skbio/diversity/alpha/_base.py

```diff
@@ -847,6 +848,51 @@ def osd(counts):
     counts = _validate_counts_vector(counts)
     return observed_otus(counts), singles(counts), doubles(counts)
 
+@experimental(as_of="0.4.0")
```

Comment:
`pip install flake8` there may be other packages that it will need in the same way. They should all be pip installable.